test/integration/meson.build

```
@@ -1,7 +1,8 @@
 if get_option('enable_tests')
   tests = [
     'open_analyse_save_load_project',
-    'bin_vfiles'
+    'bin_vfiles',
+    'cpu_platform_profiles'
```

Not important, but I think this should be valid in meson.build files. This way you don't have to modify the last line whenever you add a new entry to the array
```suggestion
    'cpu_platform_profiles',
```